src/tests/unit_tests/test_util.py

```diff
@@ -189,3 +194,46 @@ def test_extract_numbers_from_text_fails(source_data, incorrect_result_data):
     # test aganist a text which don't have the same numers as the source
     check_result = check_if_number_exist(incorrect_result_data, source_data)
     assert not check_result["number_exists"]
+
+
+def test_entity_comparator():
+    word1 = "ethiopian"
+    word2 = "ethiopia"
+    lev_constant = 0.5
+
+    lists_of_test = [
+        ["Ethiopia", "Ethiopian"],
+        ["Gastrointestinal", "astrointestin"],
+        ["World Cup", "World football Cup"],
+        ["New York City", "New York"],
+        ["New York", "New York ****"],
+    ]
+    print('#########################')
+    print('#########################')
+    for tester in lists_of_test:
+        string_comparison_with_jaccard_and_levenshtein(
+            tester[0], tester[1], lev_constant
+        )
+    assert True
```

Comment:
This test as written is not useful; it will only fail if the string comparison function raises an exception. 

Instead, 
- make assertions about the correctness of the comparisons, for each item in `lists_of_test`
- Use parameterized tests @pytest.mark.parametrize
- Test the happy path and the failure path